### ISSUE TITLE
Getting 403 Forbidden when trying to upload video to the input s3 bucket as an authenticated user

### ISSUE THREAD

**Issue by pedramp20** [awslabs/amplify-video#316]
**Describe the bug**
I followed the instruction to setup VOD and configured it to allow authenticated users to be able to upload to the input s3 bucket. When I check the AuthRole it has the policy allowing PutObject to the input bucket. however, I get forbidden 403 error when trying to upload a video file to the bucket. I am using the code provided [here](https://github.com/awslabs/amplify-video/wiki/Uploading-Assets). Has anybody else faced the same issue? Is it a limitation of amplify to support multiple buckets?

**Comment by nathanagez** [comment]
Hi @pedramp20, does your user is part of Cognito's Admin group ?

**Comment by pedramp20** [comment]
@nathanagez no it is not. As I explain, I chose the option where any authenticated user can upload videos. 

```
? Choose what project you want to update? fitooVod
? Select a system-provided encoding template, specify an already-created template name:  Default HLS Adaptive Bitrate
? Do you want to get notifications on the video processing job? Yes
? Do you want a custom function executed for notifications? Yes
? Is this a production enviroment? Yes
? We detected you have signed urls configured. Would you like to: Leave as configured
? Do you want Amplify to create a new GraphQL API to manage your videos? (Beta) Yes
Using fitoo to manage API
? Define your permission schema (Press <space> to select, <a> to toggle all, <i> to invert selection, and <enter> to proceed)
 ( ) Admins can only upload videos
>(*) Any authenticated user can upload videos
```

Based on my understanding and as the name suggests any authenticated user should be able to upload video? Am I correct @nathanagez ?

**Comment by pedramp20** [comment]
Thank you for your help. As you pointed out, when there are other groups with lower precedence, the user assumes that role and since the policy with PutObject permission is not added those roles, user gets 403 when calls the s3 api. I think this is very important to include in the documentation. 
The option provided in the CLI only adds the policy to the AuthRole and if other roles exist, it is pretty much useless. 
The approach of hard coding a role in the code is not very helpful either, while this option could be given to user to tick the groups s/he wants to grant the permission.
As a solution, I am trying to pass the userpoolId to the template to add the policy to those roles and based on your documentation, all the files with the same name in the custom folder is replaced with the items in the build folder but it is not what I am expreincing.
@nathanagez  can you please confirm if this is the case?

Basically I have added the parameter to the  paramters.json, props.jason, the main vod template and s3input template but on push they are not copied to the build folder and as a result I get an error stating "Parameters: [UserPoolId] do not exist in the template
